pkgs/development/compilers/dasm/default.nix

```
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub}:
```

You don't consider my suggestions nits? Well then, here's a nit for ya:

```suggestion
{ stdenv, fetchFromGitHub }:
```

(please :) )

---

lol. Nope! bring it on :)

ahh. I just discovered `nixfmt`, too.

---

Yeah, nixfmt is cool but it's not official yet. I.e. it's not the standard for nixpkgs.